lib/flipper/ui.rb

```diff
@@ -45,7 +45,7 @@ def self.app(flipper, options = {})
       builder.use Middleware, flipper
       builder.run app
       klass = self
-      builder.define_singleton_method(:inspect) { klass } # pretty rake routes output
+      builder.define_singleton_method(:inspect) { klass.to_s } # pretty rake routes output
```

Comment:
Maybe it would make more sense to call `inspect` instead of `to_s` here, to set this up as a more straight-forward delegation? The output is the same though.
```rb
builder.define_singleton_method(:inspect) { klass.inspect } # pretty rake routes output
```